### PR TITLE
Version example, remaining bugfixes

### PR DESCRIPTION
**Pull request by grechaw**
Hi @bsrikan Here is the version example, with reasonable docs and some refactoring.
I think it is all in good shape.  There's a lot here though so if you'd like to step through with me that would work well.

### CODE REVIEW

**Comment by bsrikan** [comment]
@grechaw, I havent got around to take a look at it yet. Will do so this week though.

File: entity-services-examples/example-races/races-qc.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@ let $race-e := race:extract-instance-Race(
   doc("/entity-services-examples/example-races/data/race-data/races/10k2.json")
 )
 let $race-f := race:extract-instance-Race(
-  doc("/entity-services-examples/example-races/data/race-data/races/10k1.json")
+  doc("/entity-services-examples/example-racels/data/race-data/races/10k1.json")
```
Comment (bsrikan) [inline entity-services-examples/example-races/races-qc.xml]: @grechaw Is this intentional "racels" vs "races" or typo.
Also I see that database dropdown doesnt default to entity-services-examples-content anymore.

File: entity-services-examples/gradle.properties [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # these defaults assume a fresh system running on your
 # local computer, which you secured with admin/admin credentials
 # (essentially, no security)
-mlHost=localhost
+mlHost=f23-runner
```
Comment (bsrikan) [inline entity-services-examples/gradle.properties]: @grechaw Guess mlHost should point back to localhost

File: entity-services-examples/src/main/resources/application.properties [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # These values typically do not need to be changed
 # They align with the users created with mlDeploy
-mlHost=localhost
+mlHost=f23-runner
```
Comment (bsrikan) [inline entity-services-examples/src/main/resources/application.properties]: @grechaw Guess mlHost should point back to localhost

**Comment by bsrikan** [comment]
Have reviewed the changes in this PR and added comments. Running example scenarios and functional tests to verify the changes.

**Comment by grechaw** [comment]
Thanks for the look-- definitely I need to switch configs to localhost, and yes 'racels' does look like a typo

**Comment by bsrikan** [comment]
I am not able to import races-qc.xml. Getting an error as:
```
Workspace Import Failed. 

Please verify that the contents of the Workspace are valid, fix any problems and retry the import.
```

Was trying on Feb28 build. Installing todays nightly to see if it persists.

**Comment by grechaw** [comment]
I was able to import just now.  Build from last night.

**Comment by grechaw** [comment]
Pushed a commit that fixes typo in qc workspace and the host names.

**Comment by grechaw** [comment]
Assigned all bugs covered by this PR to @bsrikan

File: entity-services-examples/example-versions/README.md [hunk 4]
```diff
@@ -143,13 +143,18 @@ they must be managed as part of the evolution of a hub.
...
+You may have noticed that example changes data simply by adding a property to the Person type.  It also makes a change
+by adding a word lexicon on that property.  An index change is a change in database state, and so it is either done or not
 done.  Once the index is configured and deployed, applications can use it.  If a model were to change such
 that indexes are no longer needed, the system must maintain both indexes until the data migration is complete.
 In this scenario, we've chosen simply to include the "Model-next" index configuration in the deployment.  Its
 presence does not affect the original hub, and so during the course of this exercise it is latent, only used
 by the 'next' version of search options.
 
+Something similar stands for extraction templates.  Since we wanted views of both kinds of instances to be active at
+once, the extraction templates for each model have unique Schema Names.  This means that 'Model.Peron' and 'ModelNex.Person'
```
Comment (bsrikan) [inline entity-services-examples/example-versions/README.md]: typo 'Model.Peron' and 'ModelNex.Person'

**Comment by bsrikan** [comment]
@grechaw I see that ./gradlew runExampleVersions is running some query and some results are generated. Can you include this command as a step in README. It is not clear that example-versions do not need version-qc.xml. 
It was not clear how "Search for id:1" always had different results, I assume it is referring to a different fn:doc() or collection maybe. Can you please calrify?
It will be great IMO if you can add the response that runExampleVersion returns to README so its easier to map the output alongside the graphs

**Comment by grechaw** [comment]
OK, right, we had spoken about including these responses and explanations in the README.
I'll open a new bug for that.

**Comment by bsrikan** [comment]
@grechaw Good to merger. Updates to README can be handled in a new bug as you have said.

**Comment by grechaw** [comment]
Thanks Srikanth!

**Pull request merged**